src/main/frontend/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["Vue.volar", "Vue.vscode-typescript-vue-plugin"]
```

Comment:
Why the exclusion for this one .vscode file?

Reply:
No idea. Everything in src/main/frontend was autogenerated by `npm init vue@latest`, and all except for public/favicon.ico and package.json were left unmodified by me.